### PR TITLE
Add morgan based logging

### PR DESCRIPTION
**Pull request by SivanMehta**
~Not really sure how to test this, so any advice on that front would be great!~

### CODE REVIEW

**Comment by fritzmonkey** [comment]
For testing, you could create an instance of the service, and then mock `app.log.info`. Then, you just need to make a request to `/healthcheck`, and verify the data that `app.log.info` is called with. The app integration tests should provide a good example/starting point for this.

https://github.com/godaddy/warehouse.ai/blob/master/test/integration/app.test.js

File: package.json [hunk 1]
```diff
@@ -89,7 +91,7 @@
     "proxyquire": "~1.8.0",
     "registry-mock": "^2.0.0",
     "request": "~2.81.0",
-    "sinon": "~1.17.7",
+    "sinon": "^6.1.4",
```
Comment (fritzmonkey) [inline package.json]: We should probably update more of these. At least the dev deps as some general maintenance.

File: test/integration/app.test.js [hunk 2]
```diff
@@ -75,4 +79,51 @@ describe('App (integration)', function () {
...
+            port: 0
+          }
+        }
+      }
+    }, function (err, app) {
+      assume(err).equals(null);
+      assume(app).truthy();
+      assume(app.log).is.an('object');
+
+      sinon.stub(app.log, 'info').callsFake((...msg) => logs.push(msg));
```
Comment (fritzmonkey) [inline test/integration/app.test.js]: We should call `sinon.restore` in an `afterEach`, just to be safe.

File: test/integration/app.test.js [hunk 3]
```diff
@@ -75,4 +79,51 @@ describe('App (integration)', function () {
...
+      request(address(app, { pathname: 'healthcheck' }), function (err, res) {
```
Comment (fritzmonkey) [inline test/integration/app.test.js]: `err` is already defined in the parent scope.

File: package.json (hunk 1 above)
Comment (SivanMehta) [reply package.json]: This is an open source project, can we just get Greenkeeper to look at this repo?

**Pull request merged**